README.md

```diff
@@ -119,69 +119,67 @@ The macOS installation application is [code signed](https://developer.apple.com/
-       SHA1 fingerprint: FF 67 97 79 3A 3C D7 98 DC 5B 2A BE F5 6F 73 ED C9 F8 3A 64
+    2. Apple Code Signing Certification Authority
+       SHA1 fingerprint: 1D 01 00 78 A6 1F 4F A4 69 4A FF 4D B1 AC 26 6C E1 B4 59 46
        -----------------------------------------------------------------------------
     3. Apple Root CA
        SHA1 fingerprint: 61 1E 5B 66 2C 59 3A 08 FF 58 D1 4A E2 24 52 D1 98 DF 6C 60
-
 ```
 
 You may also use the `codesign` command to examine an application's code signature:
 
 ```shell
-$ codesign -dvv /Applications/Install\ macOS\ Sierra.app
-Executable=/Applications/Install macOS Sierra.app/Contents/MacOS/InstallAssistant
-Identifier=com.apple.InstallAssistant.Sierra
+$ codesign -dvv /Applications/Install\ macOS\ Mojave.app
+Executable=/Applications/Install macOS Mojave.app/Contents/MacOS/InstallAssistant_springboard
+Identifier=com.apple.InstallAssistant.Mojave
 Format=app bundle with Mach-O thin (x86_64)
-CodeDirectory v=20200 size=297 flags=0x200(kill) hashes=5+5 location=embedded
-Signature size=4167
-Authority=Apple Mac OS Application Signing
-Authority=Apple Worldwide Developer Relations Certification Authority
+CodeDirectory v=20100 size=274 flags=0x2000(library-validation) hashes=3+3 location=embedded
+Platform identifier=5
+Signature size=4535
+Authority=Software Signing
+Authority=Apple Code Signing Certification Authority
 Authority=Apple Root CA
-Info.plist entries=30
-TeamIdentifier=K36BKF7T3D
-Sealed Resources version=2 rules=7 files=137
-Internal requirements count=1 size=124
+Info.plist entries=34
+TeamIdentifier=not set
+Sealed Resources version=2 rules=13 files=194
+Internal requirements count=1 size=84
 ```
 
 ### Creating a bootable USB installer
 
 Instead of booting from the network or using target disk mode, a bootable macOS installer can be made with the `createinstallmedia` utility included in `Contents/Resources` folder of the installer application bundle. See [Create a bootable installer for macOS](https://support.apple.com/en-us/HT201372), or run the utility without arguments to see how it works.
 
-**Note** Apple's installer [does not appear to work](https://github.com/drduh/macOS-Security-and-Privacy-Guide/issues/120) across OS versions. If you want to build a 10.12 image, for example, the following steps must be run on macOS verison 10.12!
-
 To create a **bootable USB installer**, mount a USB drive, and erase and partition it, then use the `createinstallmedia` utility:
 
 ```shell
 $ diskutil list
-[Find disk matching correct size, usually "disk2"]
+[Find disk matching correct size, usually the last disk. i.e. /dev/disk2]
 
 $ diskutil unmountDisk /dev/disk2
 
 $ diskutil partitionDisk /dev/disk2 1 JHFS+ Installer 100%
 
-$ cd /Applications/Install\ macOS\ Sierra.app
+$ cd /Applications/Install\ macOS\ Mojave.app
 
-$ sudo ./Contents/Resources/createinstallmedia --volume /Volumes/Installer --applicationpath /Applications/Install\ macOS\ Sierra.app --nointeraction
```

Comment:
`--applicationpath` is deprecated w/ the version of `createinstallmedia` on High Sierra so, I dropped it as well:
```
❯ sudo ./Contents/Resources/createinstallmedia --volume /Volumes/Installer --applicationpath /Applications/Install macOS Mojave.app
WARNING: "--applicationpath" is deprecated in macOS 10.14 and greater. Please remove it from your invocation.
```